openjdk/excludes/ProblemList_openjdk8-openj9.txt

```diff
@@ -18,6 +18,17 @@
 
 # jdk_beans
 
+java/beans/PropertyEditor/6380849/TestPropertyEditor.java https://github.com/adoptium/aqa-tests/issues/117 macosx-all
+java/beans/PropertyEditor/TestColorClass.java https://github.com/adoptium/aqa-tests/issues/117 macosx-all
+java/beans/PropertyEditor/TestColorClassJava.java https://github.com/adoptium/aqa-tests/issues/117 macosx-all
+java/beans/PropertyEditor/TestColorClassNull.java https://github.com/adoptium/aqa-tests/issues/117 macosx-all
+java/beans/PropertyEditor/TestColorClassValue.java https://github.com/adoptium/aqa-tests/issues/117 macosx-all
+java/beans/PropertyEditor/TestFontClass.java https://github.com/adoptium/aqa-tests/issues/117 macosx-all
+java/beans/PropertyEditor/TestFontClassJava.java https://github.com/adoptium/aqa-tests/issues/117 macosx-all
+java/beans/PropertyEditor/TestFontClassNull.java https://github.com/adoptium/aqa-tests/issues/117 macosx-all
+java/beans/PropertyEditor/TestFontClassValue.java https://github.com/adoptium/aqa-tests/issues/117 macosx-all
+java/beans/XMLEncoder/java_awt_ScrollPane.java https://github.com/adoptium/aqa-tests/issues/117 macosx-all
```

Comment:
Similar reason. We should not reference the Adoptium infra issue for openj9 test failure.